API Blueprint Specification.md

```diff
@@ -79,6 +81,9 @@ All of the blueprint sections are optional. However, when present, a section **m
 
 + [`0-1` **Metadata** section](#def-metadata-section)
 + [`0-1` **API Name & overview** section](#def-api-name-section)
++ [`0-1` **Authentication** section](#def-authentication-section)
+    + [`1` **Authentication schemes** section](#def-authenticationschemes-section)
+    + [`1` **Response** section](#def-response-section)
 + [`0+` **Resource** sections](#def-resource-section)
```

Comment:
Shouldn't it be possible to specify `Authentication` on `Resource` section as well?